src/FixtureBuilder/Denormalizer/Fixture/SpecificationBagDenormalizer/SimpleSpecificationsDenormalizer.php

```diff
@@ -62,6 +62,10 @@ public function denormalize(FixtureInterface $scope, FlagParserInterface $parser
         $calls = new MethodCallBag();
 
         foreach ($unparsedSpecs as $unparsedPropertyName => $value) {
+            if (!is_string($unparsedPropertyName)) {
```

Comment:
`false ===`, I know it doesn't matter much but I prefer to keep the codebase somewhat consistent :)